cdap-ui/app/cdap/components/Home/index.js

```diff
@@ -108,18 +114,61 @@ class Home extends Component {
       //FIXME: Must redirect if passed invalid namespace
     }
   }
+
+  processQueryString(queryString) {
+    let sortIndex = queryString.indexOf('sort=desc');
```

Comment:
we should just get the sorting directly from the query parameter, don't check it. We will be adding more sorting options in the future